schemas/adaptive-card.json

```diff
@@ -91,20 +123,25 @@
 			"properties": {
 				"type": {
 					"type": "string",
-					"description": "Must be `\"Action.ShowCard\"`."
+					"description": "Must be `\"Action.ShowCard\"`.",
+					"enum": [
+						"Action.ShowCard"
+					]
 				},
 				"title": {
 					"type": "string",
 					"description": "Label for button or link that represents this action."
 				},
 				"iconUrl": {
 					"type": "string",
-					"format": "uri",
-					"description": "Optional icon to be shown on the action in conjunction with the title",
+					"description": "Optional icon URL to be shown on the action in conjunction with the title. Supports data URI in version 1.2+.",
```

Comment:
>URL [](start = 35, length = 3)

The description is inconsistent with Action.OpenUrl's description. This description on Action.ShowCard has an extra "icon URL to be shown" while in Action.OpenUrl it just says "icon to be shown". I like "icon to be shown" more.